### ISSUE TITLE
Cumulative values should NEVER be empty

### ISSUE THREAD

**Issue by apfeuti** [openZH/covid_19#459]
For example see data of canton GE or NE. There are empty fields for cumulative values.

**Problem:** A client cannot "snapshot" easily into a day. The client needs logic to lookup empty values from previous days. Interpreting empty values as zero (0) is wrong, because cumulative values can NEVER decrease.

**Expected behaviour:** If a value is not known for a particular day, the value from the previous day should be taken (instead of empty). If you want to report, that NO value was avaiable, then this should be reported in a own field. For example like this:  ncumul_conf_from_previous_day = true

**Comment by baryluk** [comment]
Duplicate of https://github.com/openZH/covid_19/issues/335 and https://github.com/openZH/covid_19/issues/309

tl;dr: Yes, you need client logic to do a lookup. You are responsible to figuring out if outdated data are ok for your application or not.


Alternative is to provide non-canonical filled files in a separate csv files.

**Comment by baryluk** [comment]
One of the serious drawback of using previous values, is that if you then plot or analyse data blindly, you will have incorrect conclusions about the situation right now. It will appear that infections are slowing down, which is not necessarily true.

**Comment by metaodi** [comment]
As @baryluk explained, this is done by design.

What you decribe is perfectly valid, but out-of-scope for this repo. It's probably a nice feature for your [REST-API](https://github.com/apfeuti/covid19-rest) (maybe with an extra parameter to carry forward if no new number is available). WDYT @apfeuti?

**Comment by apfeuti** [comment]
Thank you for clarification! Yes, I was investigating why the aggregated total of https://covid19-rest.herokuapp.com/api/openzh/v1/all was "only" something more than 17000. Then I realized, that for example GE was counted with zero. 

I will add additional fields with forwarded values on my service and use them to calculate aggregated values. I think many clients will struggle with this, because they are interested on figures for a given date (regardless if the canton reported a dedicated value for this date or not). So this could be an added value of my REST-API.

**Comment by metaodi** [comment]
👍
